demo/micropython-run.ts

```diff
@@ -5,20 +5,37 @@ import { ConsoleLogger, LogLevel } from '../src/utils/logging';
 import { bootromB1 } from './bootrom';
 import { loadUF2, loadMicropythonFlashImage } from './load-flash';
 import fs from 'fs';
+const minimist = require('minimist');
+
+const args = minimist(process.argv.slice(2), {
+  string: 'image',  // UF2 image to load; defaults to "rp2-pico-20210902-v1.17.uf2"
+  boolean: 'no_gdb',  // Do NOT start GDB server
+})
 
 const mcu = new RP2040();
 mcu.loadBootrom(bootromB1);
 mcu.logger = new ConsoleLogger(LogLevel.Error);
-loadUF2('rp2-pico-20210902-v1.17.uf2', mcu);
+
+var image_name: string;
+if(args.image){
+  image_name = args.image;
+} else {
+  image_name = 'rp2-pico-20210902-v1.17.uf2';
+}
+console.log(`Loading uf2 image ${image_name}`);
+loadUF2(image_name , mcu);
 
 if (fs.existsSync('littlefs.img')) {
   loadMicropythonFlashImage('littlefs.img', mcu);
   // Instead of reading from file, it would also be possible to generate the LittleFS image on-the-fly here, e.g. using
   // https://github.com/wokwi/littlefs-wasm or https://github.com/littlefs-project/littlefs-js
 }
 
-const gdbServer = new GDBTCPServer(mcu, 3333);
-console.log(`RP2040 GDB Server ready! Listening on port ${gdbServer.port}`);
+var gdbServer: GDBTCPServer;
```

Comment:
you can remove this line and add the word `const` before `gdbServer = ...` below

Reply:
I thought the scope of the variable might cause the server to prematurely terminate, but it seems like thats not the case 👍 .